.gitignore

```diff
@@ -1,7 +1,9 @@
 *.gem
+*.iml
 *.rbc
 .bundle
 .config
+.idea
```

Comment:
@matobet Was this include by mistake?  I am not against excluding these file types but think it should be a separate PR.